.github/workflows/build-and-test.yaml

```
@@ -589,13 +509,28 @@ jobs:
     - uses: erlef/setup-beam@v1
+      if: matrix.os != 'ubuntu-24.04-arm'
       with:
         version-type: ${{ matrix.version_type || 'loose' }}
-        otp-version: ${{ matrix.otp }}
-        elixir-version: ${{ matrix.elixir_version }}
-        rebar3-version: ${{ matrix.rebar3_version }}
-        gleam-version: ${{ matrix.gleam_version }}
+        otp-version: ${{ matrix.otp || env.DEFAULT_OTP_VERSION }}
+        elixir-version: ${{ matrix.elixir_version || env.DEFAULT_ELIXIR_VERSION }}
+        rebar3-version: ${{ matrix.rebar3_version || env.DEFAULT_REBAR3_VERSION }}
+        gleam-version: ${{ matrix.gleam_version || env.DEFAULT_GLEAM_VERSION }}
```

actionlint (1.7.11)

[error] property "gleam_version" is not defined in object type {arch: string; cc: string; cflags: string; cmake_opts_other: string; compiler_pkgs: string; container: string; cxx: string; elixir_version: string; jit_target_arch: string; library-arch: string; os: string; otp: string; rebar3_version: string; version_type: string}

(expression)